.vscode/launch.json

```diff
@@ -0,0 +1,35 @@
+{
```

Comment:
why is this file added?

Reply:
This allows for debugging without the vs code extension, which has been pretty spotty for me lately